app/controllers/courses_controller.rb

```diff
@@ -28,6 +28,21 @@ def self.registered(app)
 
             @course_coll = app.settings.courses_db.collection("courses#{params[:semester]}")
             @section_coll = app.settings.courses_db.collection("sections#{params[:semester]}")
+            @prof_coll = app.settings.courses_db.collection("profs#{params[:semester]}")
+          end
+
+
+          # Route for professors, use 'name' or 'course' to filter
+          app.get '/v0/courses/professors' do
```

Comment:
do professors exist outside of courses? if so, it might make sense for `professors` or `instructors` to be their own endpoint. e.g: `/v0/instructors`